lib/main_common.pm

```diff
@@ -298,13 +298,13 @@ sub is_leanos {
 
 sub is_desktop_module_selected {
     # desktop applications module is selected if following variables have following values:
-    # productivity and ha require desktop applications, so it's preselected
+    # ha require desktop applications, so it's preselected
     # same is true for sles4sap
     return
          get_var('ADDONS', '') =~ /all-packages|desktop|we/
       || get_var('WORKAROUND_MODULES', '') =~ /desktop|we/
       || get_var('ADDONURL',           '') =~ /desktop|we/
-      || get_var('SCC_ADDONS',         '') =~ /desktop|we|productivity|ha/
+      || get_var('SCC_ADDONS',         '') =~ /desktop|we|ha/
```

Comment:
HA doesn't need Desktop Applications anymore, so you could either remove it in this PR, or I will do it after.